packages/react-scripts/config/rangle/postcss.js

```diff
@@ -0,0 +1,32 @@
+const webpack = require('webpack');
+
+const postcssBasePlugins = [
+  require('postcss-modules-local-by-default'),
+  require('postcss-import')({
+    addDependencyTo: webpack,
+  }),
+  require('postcss-cssnext')({
+    browsers: [
```

Comment:
That is actually a good idea. I couldn't find any setup for that on the cssnext docs. I believe you're referring to a customization like I did for jest? (sort of)